src/EditorFeatures/CSharp/Outlining/Outliners/DocumentationCommentOutliner.cs

```diff
@@ -35,12 +37,35 @@ private static string GetBannerText(DocumentationCommentTriviaSyntax documentati
             string text;
             if (summaryElement != null)
             {
-                var summaryText = from xmlText in summaryElement.ChildNodes().OfType<XmlTextSyntax>()
-                                  from tk in xmlText.TextTokens
-                                  let s = tk.ToString().Trim()
-                                  where s.Length > 0
-                                  select s;
-                text = prefix + " <summary> " + string.Join(" ", summaryText);
+                var sb = new StringBuilder(summaryElement.Span.Length);
+                sb.Append(prefix);
+                sb.Append(" <summary>");
+                foreach (var node in summaryElement.ChildNodes())
+                {
+                    if (node is XmlTextSyntax)
+                    {
+                        var textTokens = ((XmlTextSyntax)node).TextTokens;
+                        AppendTextTokens(sb, textTokens);
+                    }
+                    else if (node is XmlEmptyElementSyntax)
+                    {
+                        var e = (XmlEmptyElementSyntax)node;
+                        var cref = e.Attributes.OfType<XmlCrefAttributeSyntax>().FirstOrDefault();
+                        if (cref != null)
```

Comment:
I know this bug is about crefs, but we want to do paramrefs and typeparamrefs too?